scripts/initStudy.bash

```diff
@@ -23,6 +26,7 @@ echo "
 ## Next Steps
 
 - cd '$1'
+- npm install --progress
```

Comment:
I tried to from the shell script, using npm install --no-progress.  Something about how stdout gets managed made it seem weird and hanging.  Opening as a new bug. #20